tutorials/LinearAlgebra/LinearAlgebra.ipynb

```diff
@@ -0,0 +1,983 @@
+    "    for i in range(n):\n",
+    "        # Loops can be nested\n",
+    "        for j in range(m):\n",
+    "            # You can access elements of a matrix like this\n",
+    "            x = a[i][j]\n",
+    "            \n",
+    "            # You can also modify the elements of a matrix\n",
+    "            c[i][j] = ...\n",
+    "    \n",
+    "    return c"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Scalar Multiplication\n",
+    "\n",
+    "Another rather simple matrix operation is **scalar multiplication** - multiplying the entire matrix by a scalar (number):\n",
+    "\n",
+    "$$a \\cdot\n",
+    "\\begin{bmatrix}\n",
+    "    x_{0,0} & x_{0,1} & \\dotsb & x_{0,m-1} \\\\\n",
+    "    x_{1,0} & x_{1,1} & \\dotsb & x_{1,m-1} \\\\\n",
+    "    \\vdots  & \\vdots  & \\ddots & \\vdots  \\\\\n",
+    "    x_{n-1,0} & x_{n-1,1} & \\dotsb & x_{n-1,m-1}\n",
+    "\\end{bmatrix}\n",
+    "=\n",
+    "\\begin{bmatrix}\n",
+    "    a \\cdot x_{0,0} & a \\cdot x_{0,1} & \\dotsb & a \\cdot x_{0,m-1} \\\\\n",
+    "    a \\cdot x_{1,0} & a \\cdot x_{1,1} & \\dotsb & a \\cdot x_{1,m-1} \\\\\n",
+    "    \\vdots  & \\vdots  & \\ddots & \\vdots  \\\\\n",
+    "    a \\cdot x_{n-1,0} & a \\cdot x_{n-1,1} & \\dotsb & a \\cdot x_{n-1,m-1}\n",
+    "\\end{bmatrix}$$\n",
+    "\n",
+    "The scalar may be real or complex.\n",
+    "\n",
+    "Scalar multiplication has the following properties:\n",
+    "\n",
+    "* Associativity: $x \\cdot (yA) = (x \\cdot y)A$\n",
+    "* Distributivity over matrix addition: $x(A + B) = xA + xB$\n",
+    "* Distributivity over scalar addition: $(x + y)A = xA + yA$"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### <span style=\"color:blue\">Exercise 2</span>: Scalar multiplication\n",
+    "\n",
+    "**Inputs:**\n",
+    "\n",
+    "1. A scalar $x$.\n",
+    "2. An $n \\times m$ matrix $A$.\n",
+    "\n",
+    "**Goal:** Return the matrix that results from the multiplication $x \\cdot A$."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "@exercise\n",
+    "def scalar_mult(x : complex, a : Matrix) -> Matrix:\n",
+    "    # Fill in the missing code and run the cell to check your work.\n",
+    "    return # ..."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Matrix Multiplication\n",
+    "\n",
+    "A more complicated, but much more important and interesting operation is **matrix multiplication**. The first complicated thing about it is that neither its operands nor its output are the same size: an $n \\times m$ matrix multiplied by an $m \\times k$ matrix results in an $n \\times k$ matrix. That is, for matrix multiplication to be applicable, the number of columns in the first matrix must equal the number of rows in the second matrix.\n",
```

Comment:
What is the second complicated thing? Generally I'd try to steer away from the use of the word "complicated" here to avoid scaring people, since it's not really that complicated. Could you rewrite it in some less intimidating terms?